src/devices/Media/VideoDevice/Devices/UnixVideoDevice.cs

```diff
@@ -0,0 +1,509 @@
+            {
+                Settings.Contrast = GetVideoDeviceValue(VideoDeviceValueType.Contrast).DefaultValue;
+            }
+
+            if (Settings.Gain.Equals(default))
+            {
+                Settings.Gain = GetVideoDeviceValue(VideoDeviceValueType.Gain).DefaultValue;
+            }
+
+            if (Settings.Gamma.Equals(default))
+            {
+                Settings.Gamma = GetVideoDeviceValue(VideoDeviceValueType.Gamma).DefaultValue;
+            }
+
+            if (Settings.Rotate.Equals(default))
+            {
+                Settings.Rotate = GetVideoDeviceValue(VideoDeviceValueType.Rotate).DefaultValue;
+            }
+
+            if (Settings.WhiteBalanceTemperature.Equals(default))
+            {
+                Settings.WhiteBalanceTemperature = GetVideoDeviceValue(VideoDeviceValueType.WhiteBalanceTemperature).DefaultValue;
+            }
+
+            if (Settings.ColorEffect.Equals(default))
+            {
+                Settings.ColorEffect = (ColorEffect)GetVideoDeviceValue(VideoDeviceValueType.ColorEffect).DefaultValue;
+            }
+
+            if (Settings.PowerLineFrequency.Equals(default))
+            {
+                Settings.PowerLineFrequency = (PowerLineFrequency)GetVideoDeviceValue(VideoDeviceValueType.PowerLineFrequency).DefaultValue;
+            }
+
+            if (Settings.SceneMode.Equals(default))
+            {
+                Settings.SceneMode = (SceneMode)GetVideoDeviceValue(VideoDeviceValueType.SceneMode).DefaultValue;
+            }
+
+            if (Settings.WhiteBalanceEffect.Equals(default))
+            {
+                Settings.WhiteBalanceEffect = (WhiteBalanceEffect)GetVideoDeviceValue(VideoDeviceValueType.WhiteBalanceEffect).DefaultValue;
+            }
+
+            if (Settings.HorizontalFlip.Equals(default))
+            {
+                Settings.HorizontalFlip = Convert.ToBoolean(GetVideoDeviceValue(VideoDeviceValueType.HorizontalFlip).DefaultValue);
+            }
+
+            if (Settings.VerticalFlip.Equals(default))
+            {
+                Settings.VerticalFlip = Convert.ToBoolean(GetVideoDeviceValue(VideoDeviceValueType.VerticalFlip).DefaultValue);
+            }
+        }
+
+        private void Initialize()
+        {
+            if (_deviceFileDescriptor >= 0)
+            {
+                return;
+            }
+
+            string deviceFileName = $"{DevicePath}{Settings.BusId}";
+            lock (s_initializationLock)
+            {
+                if (_deviceFileDescriptor >= 0)
+                {
+                    return;
+                }
+                _deviceFileDescriptor = Interop.open(deviceFileName, FileOpenFlags.O_RDWR);
+
+                if (_deviceFileDescriptor < 0)
+                {
+                    throw new IOException($"Error {Marshal.GetLastWin32Error()}. Can not open video device file '{deviceFileName}'.");
+                }
+            }
+        }
+
+        private void Close()
```

Comment:
should this be happening only in Dispose?

Reply:
It happens at the end of each capture.